unitTest/@testFEMComputationalFluidDynamicsAnalysis/testFEM4NavierStokesSteadyStateFlowAroundCylinder2D.m

```diff
@@ -0,0 +1,138 @@
+function testFEM4NavierStokesSteadyStateFlowAroundCylinder2D(testCase)
+%% Licensing
+%
+% License:         BSD License
+%                  cane Multiphysics default license: cane/license.txt
+%
+% Main authors:    Marko Leskovar
+% Andreas Apostolatos
+%
+%% Function documentation
+%
+% Tests the stabilized finite element analysis for the 2D steady-state
+% Navier-Stokes equations, over the flow around a cylinder problem with 
+% respect to reference paper by:
+% http://www.mathematik.tu-dortmund.de/lsiii/cms/papers/SchaeferTurek1996.pdf
+%
+% Function layout :
+%
+% 0. Read input
+%
+% 1. Parse the data from the GiD input file
+%
+% 2. GUI
+%
+% 3. Choose the equation system solver
+%
+% 4. Define the name of the vtk file from where to resume the simulation
+%
+% 5. Change the input velocity to match the reference paper - parabolic input
+%
+% 6. Solve the CFD problem
+%
+% 7. Calculate drag and lift force from the nodal forces
+%
+% 8. Calculate drag and lift coefficient based on drag and lift force
+%
+% 9. Define the expected solutions
+%
+% 10. Verify the results
+%
+%% Function main body
+
+%% 0. Read input
+% Define absolute tolerance
+absTol = 0.04; % tolerance w.r.t the reference paper
+absTol2 = 1e-10; % tolerance w.r.t our value
+
+% Define the path to the case
+pathToCase = '../../inputGiD/FEMComputationalFluidDynamicsAnalysis/';
+caseName = 'unitTest_testFEM4NavierStokesSteadyStateFlowAroundCylinder2D';
+
+%% 1. Parse the data from the GiD input file
+[fldMsh,homDBC,inhomDBC,valuesInhomDBC,nodesALE,~,analysis,parameters,...
+    propNLinearAnalysis,propFldDynamics,gaussInt,~] = ...
```

Comment:
This unit test function should work with the new workflow, meaning the new state of the problem type and the parser. Therefore, postProc needs to be added to the list of the output arguments and the input file of the corresponding case updated. As a convention, the name of the input file for this case should start with unitTest_ and then the name as you had it.